ApiEverest/CrudApi/Controllers/CostumerController.cs

```diff
@@ -1,25 +1,29 @@
-﻿using ApiEverest.Entities;
-using CustomerApi.Services;
+﻿using AppServices.Services;
+using AutoMapper;
+using DomainModels.Entities;
+using DomainServices.Services;
```

Comment:
```suggestion
﻿using AppServices.Services;
using DomainModels.Entities;
```

Para facilitar a remoção dos using não utilizados usamos o comando CTRL + R + G